test/functional/api/cas/cache.py

```diff
@@ -65,7 +65,7 @@ def get_occupancy(self):
         return self.get_cache_statistics()["occupancy"]
 
     def get_status(self):
-        status = self.get_cache_statistics()["status"].replace(' ', '_')
+        status = self.get_cache_statistics()["status"].replace(' ', '_').lower()
```

Comment:
If PR #218 gets merged this line should look like this:
`status = self.get_statistics().config_stats.status.replace(' ', '_').lower()`